internal/testprotos/desc_test_comments.proto

```diff
@@ -83,6 +82,7 @@ message /* detached message name */ /* request with a capital R */ Request // tr
 		HEY_HO = 7;
 		MAGIKOOPA = 8;
 		KAMEK = 8;
+		SNIFIT = -101;
```

Comment:
This negative value revealed the panic when running tests for the protoprint package and confirms that the proposed fix does resolve the error.